EcoSoapBank/Model Controller/Mock Providers/MockPickupProvider.swift

```diff
@@ -189,7 +189,7 @@ extension Pickup.ScheduleInput {
     static func random() -> Self {
         Pickup.ScheduleInput(
             base: .random(),
-            propertyID: UUID(),
+            propertyID: "0",
```

Comment:
FYI, the format of id's will be TypeId. So this one would be "PickupId1" or something like that.